src/p11.c

```diff
@@ -160,7 +161,7 @@ int p11_gen_key(p11_context *p11, p11_key_ctx *p11_key, const char *label) {
             goto found;
         }
     }
-    UM_LOG(WARN, "did not find suitable key generation mechanism: %d/%s", rv, p11_strerror(rv));
+    UM_LOG(WARN, "did not find suitable key generation mechanism");
```

Comment:
nothing useful was getting logged i presume?

Reply:
yes and `rv` was not set before the log